CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 ---------
 
+**UNRELEASED**
```

Comment:
I added your previous PR as version `3.1.0`. Please rebase on the new master and instead of `UNRELEASED` the version would be `3.2.0` as we just adding a feature and do not break existing stuff.

Reply:
Cool, it has been done. Thanks!